pkg/controller/mizar/mizar-pod-controller.go

```diff
@@ -219,10 +219,19 @@ func (c *MizarPodController) handle(keyWithEventType KeyWithEventType) error {
 			klog.Warningf("Failed to retrieve network in local cache by tenant %s, name %s: %v", tenant, defaultNetworkName, err)
 			return err
 		}
-		klog.Infof("Get network: %#v.", network)
+		klog.V(4).Infof("Get network: %#v.", network)
 
-		if network.Spec.Type != mizarNetworkType || network.Status.Phase != arktosextv1.NetworkReady {
-			klog.Warningf("The arktos network %s is not mizar type or is not Ready.", network.Name)
+		if network.Spec.Type != mizarNetworkType {
+			return nil
+		}
+
+		if network.Status.Phase != arktosextv1.NetworkReady {
+			klog.Warningf("The arktos network %s is not Ready.", network.Name)
+			// put key back into queue
+			go func() {
```

Comment:
this code seems not to put it in queue but instead create it, though in non-blocking way

Reply:
same concern as hongwei's comment

Reply:
the createObj function is the add object handler of pod informer. It implemented as following:

```
func (c *MizarPodController) createObj(obj interface{}) {
	key, _ := controller.KeyFunc(obj)
	c.queue.Add(KeyWithEventType{Key: key, EventType: EventType_Create})
}
```

Reply:
thanks for clarification. so the code seems, based on the reading, when network state is of not ready yet - will put the key back to queue, so that later handling will pick it u to process again. The logic makes sense. Two minor suggestions, then:
1. createObj func name seems not express the intention fully; 
2. it might not need to go async; simple call seems ok

These are not blocking the PR; for author's considerations only.

Reply:
> 

. Carl will bring the code change to production. I am refraining from making too much code changes. Also in deployment controller, the add event handler called addDeployment. So it seems ok for the even handler naming.

. Immediately putting the object back into queue could be busy waiting. Putting into async so that this particular pod won't take the worker when it sleep since workers are limited.